Assets/MixedRealityToolkit.Examples/Demos/StandardShader/New feature/MaterialParameterCecker.cs

```diff
@@ -0,0 +1,374 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using UnityEngine.UI;
+using Microsoft.MixedReality.Toolkit.UI;
+
+
+public class MaterialParameterCecker : MonoBehaviour
```

Comment:
please provide a class description with /// comment so users can read up on it in the API documentation